roles/openshift_logging_kibana/tasks/main.yaml

```diff
@@ -141,7 +143,9 @@
     dest: "{{ tempdir }}/templates/oauth-client.yml"
   vars:
     kibana_hostname: "{{ openshift_logging_kibana_hostname }}"
+    kibana_ops_hostname: "{{ openshift_logging_kibana_ops_hostname }}"
```

Comment:
we shouldn't need to do this, if the openshift_logging_kibana_hostname is incorrect being passed into the role we should adjust that at the `openshift_logging/tasks/install_logging.yml` level. It breaks the pattern being used